pkg/program/shell/run_dir_unix.go

```diff
@@ -31,7 +32,19 @@ func getSecureRunDir() (string, error) {
 		return "", err
 	}
 
-	return runDir, checkExclusiveAccess(info, uid)
+	return runDir, checkExclusiveAccess(info, os.Getuid())
+}
+
+// getRunDirPath returns a path that's used by elvish to store runtime files.
```

Comment:
Nitpick: Unless referring to the command name or file path, spell elvish with a capital E.